docs/guide/core-concepts.rst

```diff
@@ -13,8 +13,8 @@ alike. If GIS is brand, brand new to you, `this
 document <https://www.gislounge.com/what-is-gis/>`__ is a useful high
 level overview.
 
-Basic Terms
-===========
+Glossary
+========
```

Comment:
I think it might be helpful to have links to each of the sections where these topics are discussed. That way you don't have to spend a lot of time trying to find it.